src/pages/index.js

```diff
@@ -103,30 +86,23 @@ function handleEditProfileSubmit(formData) {
   modalEditForm.setSubmitText(true);
   api
     .editProfileData(formData)
-    .then((response) => {
-      return response.json();
-    })
     .then((data) => {
       userInfo.setUserInfo(data);
       modalEditForm.close();
     })
-    .catch((error) => {
-      console.error(error);
-    })
+    .catch(console.error)
+
     .finally(() => modalEditForm.setSubmitText(false));
 }
 
 function handleEditProfilePictureSubmit(data) {
-  profilePictureImage.src = data.url;
-  editProfilePicture.setSubmitText(true);
   api
     .updateProfilePicture(data.url)
     .then(() => {
+      profilePictureImage.src = data.url;
```

Comment:
`      profilePictureImage.src = data.url;
`

Class `UserInfo` should set the entire user’s data (including the **avatar**)